remote/scheduler/remote.go

```diff
@@ -80,10 +73,29 @@ func (c *Client) Read(inputMsgs chan<- models.Message, rules map[string]models.R
 				message.OutputToRooms = outputRooms
 				message.OutputToUsers = outputUsers
 				inputMsgs <- message
-			})
+			}
+
+			// use our logger for cron
+			cronLogger := cron.PrintfLogger(&bot.Log)
+
+			// check if the provided schedule is of standard format, ie. 5 fields
+			_, err := cron.ParseStandard(rule.Schedule)
+			if err == nil {
+				// standard cron
+				job = cron.New(cron.WithChain(cron.SkipIfStillRunning(cronLogger)))
```

Comment:
just calling out this part (`SkipIfStillRunning`) as it may be too opinionated. it seemed like a decent, sensible default, but i can be convinced to take it out or control it via flag as there might be use-cases.